src/md/basic/foundations/hello-world.md

```diff
@@ -3,68 +3,174 @@ layout: basic.11ty.js
 title: serverless web dev training with architect
 ---
 
-# Hello world 
+# Hello world
 
-Begin.com supports building static web apps alongside the popular backend JS runtimes Node and Deno. You can even mix all these things in the same app. 
+[Begin.com](https://begin.com/) supports building static web applications alongside the popular backend JavaScript runtimes Node and Deno. You can even use both Node and Deno in the same app. Or other programming languages like Ruby and Python.
```

Comment:
this is true for Architect but we currently do not support Python and Ruby apps in Begin just yet! Should remove that part